google/api_core/grpc_helpers.py

```diff
@@ -218,7 +227,7 @@ def _create_composite_credentials(credentials=None, scopes=None, ssl_credentials
     )
 
 
-def create_channel(target, credentials=None, scopes=None, ssl_credentials=None, **kwargs):
+def create_channel(target, credentials=None, scopes=None, ssl_credentials=None, credentials_file=None, **kwargs):
```

Comment:
I added `credentials_file` to the end of the list to avoid breaking anyone who is using positional arguments.